src/__a11y__/to-validate-a11y.ts

```diff
@@ -22,6 +22,7 @@ const htmlValidator = new HtmlValidate({
     // set relaxed to exclude error that id does not begin with letter
     'valid-id': ['error', { relaxed: true }],
     'no-inline-style': 'off',
+    'attribute-boolean-style': 'off',
```

Comment:
This line already got added in another PR: https://github.com/cloudscape-design/components/pull/365/files?w=1#diff-73b99913627c86b05597c2e64cdb26164b755695effee4508bf0da405b426a0eR30

Reply:
Thanks, I will rebase 👍